tests/ipfwd/test_nhop_count.py

```diff
@@ -267,11 +267,19 @@ def test_nhop(request, duthost, tbinfo):
     - Add IP route and nexthop
     - check CRM resource
     - clean up
-    - Verify no erros and crash
+    - Verify no errors and crash
     """
     skip_release(duthost, ["201811", "201911"])
 
-    default_max_nhop_paths = 32
+    # Set of parameters for Cisco-8000 devices
+    if is_cisco_device(duthost):
+        default_max_nhop_paths = 10
```

Comment:
Why don't you set to 12 to use out of all 192K member entires?

Reply:
This was failing with 12 nexthop group members because the available members at the beginning of the script keep varying. Also, because the scope of this test seems to be to exhaust all the groups and not the group members, we decided to go ahead with 10